### PR TITLE
bootflow_2711.md - minor grammar fixups

### PR DESCRIPTION
**Pull request by andrum99**
Also correct heading level.

### CODE REVIEW

File: hardware/raspberrypi/bootmodes/bootflow_2711.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Raspberry Pi4, Pi400 and CM4 bootflow
+# Raspberry Pi 4, 400 and CM4 bootflow
```
Comment (lurch) [inline hardware/raspberrypi/bootmodes/bootflow_2711.md:1]: If you're splitting it up like this, it probably ought to be "Raspberry Pi 4B, Raspberry Pi 400 and CM4 bootflow" ?

File: hardware/raspberrypi/bootmodes/bootflow_2711.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-This page describes the bootflow for BCM2711 based products. The main difference betweeen this and previous products is that the second stage bootloader is loaded from an SPI flash [EEPROM](../booteeprom.md) on BCM2711 instead of the bootcode.bin file on previous chips.
+This page describes the bootflow for BCM2711-based products. The main difference betweeen this and previous products is that the second stage bootloader is loaded from an SPI flash [EEPROM](../booteeprom.md) instead of the bootcode.bin file on previous products.
```
Comment (lurch) [inline hardware/raspberrypi/bootmodes/bootflow_2711.md:3]: Could you add the missing backticks to `bootcode.bin` please?

File: hardware/raspberrypi/bootmodes/bootflow_2711.md [hunk 3]
```diff
@@ -67,8 +67,7 @@ Please see the [bootloader configuration](../bcm2711_bootloader_config.md) page
             * Success - run the firmware
             * Failure - continue            
    * else if boot-mode == `RPIBOOT` then
-      * Attempt to load firmware using USB device mode from the USB OTG port- see [usbboot](https://github.com/raspberrypi/usbboot).
-        There is no timeout for `RPIBOOT` mode.
+      * Attempt to load firmware using USB device mode from the USB OTG port- see [usbboot](https://github.com/raspberrypi/usbboot). There is no timeout for `RPIBOOT` mode.
```
Comment (lurch) [inline hardware/raspberrypi/bootmodes/bootflow_2711.md:70]: Space before the hyphen?

File: hardware/raspberrypi/bootmodes/bootflow_2711.md [hunk 4]
```diff
@@ -19,12 +19,12 @@ The bootflow for the ROM (first stage) is as follows:-
...
       * Fail - retry USB device boot
 
-N.B. Currently only CM4 reserves a GPIO for `nRPIBOOT`
+N.B. Currently only CM4 reserves a GPIO for `nRPIBOOT`.
 
 ## recovery.bin
 `recovery.bin` is a minimal second stage program used to reflash the bootloader SPI EEPROM image.
 
-# Second stage bootloader 
+## Second stage bootloader
```
Comment (lurch) [inline hardware/raspberrypi/bootmodes/bootflow_2711.md:27]: Perhaps there ought to be a `## First stage bootloader ` header too? :shrug:

Comment (andrum99) [reply hardware/raspberrypi/bootmodes/bootflow_2711.md:29]: OK - I've added one.

File: hardware/raspberrypi/bootmodes/bootflow_2711.md (hunk 3 above)
Comment (andrum99) [reply hardware/raspberrypi/bootmodes/bootflow_2711.md]: Good catch.

**Comment by andrum99** [comment]
Thanks lurch - that's all your review comments addressed.

**Comment by lurch** [comment]
Thanks @andrum99 :+1:

**Pull request merged**